source/index.html

```diff
@@ -1,12 +1,240 @@
+              готово.
+            </p>
+          </li>
+          <li class="advantages__item advantages__item--icon-offer">
+            <p class="advantages__description">Замените один-два приема обычной еды на наше функциональное питание.
+            </p>
+          </li>
+          <li class="advantages__item advantages__item--icon-result">
+            <p class="advantages__description">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!-- Пример -->
+    <section class="example">
+      <div class="example-wrapper">
+        <div class="example__heading-wrapper">
+          <h2 class="example__title">Живой пример</h2>
+          <p class="example__text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy
+            Slim.
+            Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16
+            часов в день. </p>
+        </div>
+        <div class="example-stats__wrapper">
+          <h3 class="visually-hidden">Статистика по похудению</h3>
+          <ul class="stats__list">
+            <div class="stats__item">
+              <p class="stats__meaning">снижение веса</p>
+              <b class="stats__number">5 КГ</b>
+            </div>
+            <div class="stats__item">
+              <p class="stats__meaning">затрачено времени</p>
+              <b class="stats__number">60 ДНЕЙ</b>
+            </div>
+          </ul>
+        </div>
+        <b class="result">Затраты на питание: 15 000 РУБ.</b>
+        <div class="example-slider">
+          <div class="slider-group-image-1">
+          </div>
+          <div class="slider-group-image-2">
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Карта -->
+    <section class="map">
+      <h2 class="visualy-hidden">Как нас найти</h2>
+      <div class="invitation invitation--inside-map">
+        <div class="invation__wrapper">
+          <h3 class="invation_title">приглашаем к сотрудничеству дилеров!</h3>
+          <p class="invation__text">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</p>
+        </div>
+
+        <div class="map__wrapper">
+          <img class="map__image" src="img/map.jpg" width="320" height="362" alt="Карта, где мы находимся">
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <footer class="page__footer">
+
+    <!-- Логотип -->
+    <section class="footer__logo">
+      <div class="logo-container-2">
+        <a class="logo-container-2__logo">
+          <img class="logo-container-2__logo-image" src="img/logo-text-phone.svg" width="101" height="18"
+            alt="Текстовый логотип Кэт энерджи">
+        </a>
+      </div>
+    </section>
+
+    <!-- Социальные сети -->
+    <section class="social">
+      <div class="social__wrapper">
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <ul class="footer-social__list">
```

Comment:
```suggestion
        <ul class="social__list">
```
Далее тоже поправить.